lib/v8debugapi.js

```diff
@@ -277,6 +280,12 @@ module.exports.create = function(logger_, config_) {
     // TODO: need test coverage for the above
     // TODO: work with debug server to come up with a more robust solution
 
+    if (breakpoint.location.line >= matchingScript.line_ends.length) {
+      return setErrorStatusAndCallback(cb, breakpoint,
+        StatusMessage.BREAKPOINT_SOURCE_LOCATION,
+        messages.INVALID_LINE_NUMBER);
```

Comment:
The [style guide](https://google.github.io/styleguide/javascriptguide.xml?showone=Code_formatting#Code_formatting) seems the allow the indentation I used, with the caveat that it prefers indent = 4 spaces while our code is indent = 2.